### PR TITLE
Add Morphis Wallet to wallet standard list

### PR DESCRIPTION
**Pull request by ChiuMungZitAlexander**

### CODE REVIEW

File: sdk/wallet-adapter/packages/wallet-kit/README.md [hunk 1]
```diff
@@ -90,3 +90,4 @@ The following wallets are known to work with the Wallet Standard:
 - **[Sui Wallet](https://docs.sui.io/devnet/explore/wallet-browser)**
 - **[Ethos Wallet](https://chrome.google.com/webstore/detail/ethos-wallet/mcbigmjiafegjnnogedioegffbooigli)**
 - **[Suiet Wallet](https://suiet.app/)**
+- **[Morphis Wallet](https://chrome.google.com/webstore/detail/morphis-wallet/heefohaffomkkkphnlpohglngmbcclhi)**
```
Comment (Jordan-Mysten) [inline sdk/wallet-adapter/packages/wallet-kit/README.md:93]: Thanks for opening a PR for this! I went and tried this and it looks like the icon isn't accessible from the web. Screenshot attached.
Screen Shot 2022-12-06 at 10 27 29 PM

Comment (ChiuMungZitAlexander) [reply sdk/wallet-adapter/packages/wallet-kit/README.md:93]: Thank you for the feedback. This bug was fixed in our latest release v0.0.1.3.

**Review by Jordan-Mysten — APPROVED** [review]

**Comment by Jordan-Mysten** [comment]
@ChiuMungZitAlexander Verified and it looks good! The PR needs to be rebased (the readme moved), but I'll merge it after that!

**Comment by ChiuMungZitAlexander** [comment]
> @ChiuMungZitAlexander Verified and it looks good! The PR needs to be rebased (the readme moved), but I'll merge it after that!

Cheers! It seems there're conflicts. Should I open a new PR?

**Pull request closed without merging**